dp-cluster-setup-utility.py

```diff
@@ -29,7 +29,12 @@
 from contextlib import closing
 from urlparse import urlparse
 from shutil import copyfile
-
+# import for CM rest client
+import cm_client
```

Comment:
Here's a major one :-) How does one get this client? And what if we are working only with HDP clusters - for e.g DLM.